src/Engines/Nunjucks.js

```diff
@@ -287,57 +287,68 @@ class Nunjucks extends TemplateEngine {
         var body = parser.parseUntilBlocks("end" + shortcodeName);
         parser.advanceAfterBlockEnd();
 
-        if (isAsync) {
```

Comment:
If I’m reading this right, this makes all Nunjucks paired shortcodes into Async shortcodes? Is this slower?